documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -127,3 +128,42 @@ If a placeholder `{}` is added to the message it will be replaced with streaming
 |`{"message": "Signal message at offset {}"}`
 
 |===
+
+
+=== Triggering an Incremental Snapshot
+*Available for MySQL, PostgresSQL, SQL Server and Db2 connectors*
+
+The type of the action is `execute-snapshot`.
+It will start an incremental snapshot operation up to the oldest available streamed event.
+
+==== Optional Connector Configuration
+`incremental.snapshot.chunk.size`: Integer value. The number of rows collected at each fetch operation on the database.
+
+
+.Action parameters
+[cols="3,9",options="header"]
+|===
+|Name | Description
+
+|data-collections
+|The list of data collections to be snapshoted. +
+For MySQL, PostgresSQL and Db2 the collections should be formatted as "some_schema.some_table" + 
+For SQL Server it should be formatted as "some_database.some_schema.some_table"
```

Comment:
A formating detail "some_database.some_schema.some_table" (and other instances) should be formatted as `"some_database.some_schema.some_table"`